modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java

```diff
@@ -296,22 +297,23 @@ private JSONObject _getExportFileFormatJSONObject(
 		);
 	}
 
-	private String _getExportTranslationURLString() {
-		LiferayPortletURL liferayPortletURL =
-			_liferayPortletResponse.createResourceURL(
-				TranslationPortletKeys.TRANSLATION);
-
-		liferayPortletURL.setResourceID("/translation/export_translation");
-
-		return PortletURLBuilder.create(
-			liferayPortletURL
-		).setParameter(
-			"classNameId", _classNameId
-		).setParameter(
-			"classPK", ArrayUtil.toStringArray(_classPKs)
-		).setParameter(
-			"groupId", _groupId
-		).buildString();
+	private String _getExportTranslationURLString() throws URISyntaxException {
+		URIBuilder.URIBuilderWrapper uriBuilderWrapper = URIBuilder.create(
+			PortalUtil.getPortalURL(_httpServletRequest) + Portal.PATH_MODULE +
+				"/translation/export_translation"
+		).addParameter(
+			"classNameId", String.valueOf(_classNameId)
+		).addParameter(
+			"groupId", String.valueOf(_groupId)
+		);
+
+		for (long classPK : _classPKs) {
```

Comment:
my only doubt is that previously we were sending only one classPK parameter with a list of classPks (if several) and now it seems that it's a list classPk params.

If this is controlled and it's the intended behavior, after the Jorge suggestion, LGTM

Reply:
It is equivalent; adding multiple values to a parameter is the same thing as adding that parameter multiple times.